### PR TITLE
Added &chain

### PR DESCRIPTION
**Pull request by Fabizocker456**
This pull was way easier than I expected. The parsing engine overhaul will have to wait until this is done.

### CODE REVIEW

File: src/commands.js [hunk 1]
```diff
@@ -167,7 +167,6 @@ Commands.comfort = new Command("Comforts the person you want", (message, args) =
 Commands.say = new Command("Says whatever you want", (message, args) => {
     if (!args.join(" ")) { args[0] = "** **" }
     message.channel.send(args.join(" ").slice(0, 1900))
-    message.delete()
```
Comment (Felix-44) [inline src/commands.js:170]: why was this line deleted?

Comment (Fabizocker456) [reply src/commands.js:170]: If the first chain-command deletes the message, and the second one tries to do anything with the message, an error is thrown
